### PR TITLE
docs: adjust readme as per docs/readme, add xud logo

### PR DESCRIPTION
**Pull request by kilrau**
Adjustments for [docs.exchangeunion.com](https://docs.exchangeunion.com) and some other minor changes.

Dependency: https://github.com/ExchangeUnion/docs/pull/49

### CODE REVIEW

**Comment by kilrau** [comment]
Updated, please give it a quick review. I tried to keep the xud logo small in size, let me know if you think I should shrink it further.

File: README.md [hunk 1]
```diff
@@ -3,26 +3,39 @@
...
 * Peer-to-peer networking with and discovery of other nodes.
-* gRPC API with web proxy to serve other applications, also accessible via the command-line interface `xucli`.
+* [gRPC API](https://grpc.io/) with web proxy to serve other applications, also accessible via the command-line interface `xucli`.
+
+## Documentation
 
-## Install
+The daemon has been designed to be as developer friendly as possible in order to facilitate application development on top of `xud`.
+* [docs.exchangeunion.com](https://docs.exchangeunion.com): the starting point and up-to-date collection of guides, links and examples
+* [api.exchangeunion.com](https://api.exchangeunion.com): the direct link to the automatically generated gRPC API documentation
```
Comment (sangaman) [inline README.md]: I think we can drop "the direct link to" in this line and below and just say "The automatically generated gRPC API documentation."

File: README.md [hunk 2]
```diff
@@ -3,26 +3,39 @@
...
+* [typedoc.exchangeunion.com](https://typedoc.exchangeunion.com/): the direct link to the automatically generated code documentation
 
-This section contains instructions for installing `xud` manually, without treating setup of dependencies like `bitcoind` or `lnd`. It is mainly geared towards developers. For all other users, we recommend our streamlined setup via [docker](https://docs.exchangeunion.com/start-trading/user-guide).
+## Installation
 
-If you already have [Node.js](https://nodejs.org/en/download/) (min 10.15.3) installed, you can install `xud` via npm by running
+This section contains brief instructions for installing `xud` manually, without treating setup of dependencies like `bitcoind` or `lnd`. It is mainly geared towards developers. For all other users, we recommend our streamlined [xud-docker](https://docs.exchangeunion.com/start-trading) setup.
+
+If you already have [Node.js](https://nodejs.org/en/download/) (latest LTS) installed, you can install `xud` via npm by running
```
Comment (sangaman) [inline README.md]: Should this say "latest or LTS" ?

File: README.md (hunk 1 above)
Comment (kilrau) [reply README.md]: Fixed

File: README.md (hunk 2 above)
Comment (kilrau) [reply README.md]: Fixed

**Pull request merged**